test/unit/org/apache/cassandra/schema/SchemaTest.java

```diff
@@ -43,19 +44,16 @@
     public static void setupDatabaseDescriptor()
     {
         DatabaseDescriptor.daemonInitialization();
+        ServerTestUtils.prepareServer();
+        Schema.instance.loadFromDisk();
```

Comment:
loadFromDisk is not exactly a DD setup, maybe it should be just a setup method and not setupDatabaseDescriptor?